react/components/Table/Toolbar.js

```diff
@@ -270,6 +357,12 @@ Toolbar.propTypes = {
     inputSearch: PropTypes.shape({
       onSubmit: PropTypes.func,
     }),
+    density: PropTypes.shape({
+      label: PropTypes.string,
```

Comment:
Maybe a better name for this "label" since there are 3 more below.

Reply:
is it better if I call it `button`? So they are all labels `button`, `low`, `medium` and `high`

Reply:
`buttonLabel` or `labelButton`?

Reply:
Did not see that you had renamed the labels below. Looks fine like this.

Reply:
Maybe `button` yes, or similar. Like `buttonLabel`